package.json

```diff
@@ -6,6 +6,7 @@
     "axios": "^0.27.2",
     "dotenv": "^16.0.1",
     "fs": "^0.0.1-security",
+    "websocket": "^1.0.34",
     "ws": "^8.8.0"
```

Comment:
Is `ws` still needed?

Reply:
Follow-up: could `ws` be used, instead? It's more popular and more recently updated.

Reply:
Uninstalled websocket module, as ws is the one I am using already.